src/lib/components/workflow/workflows-summary-configurable-table/table-empty-state.svelte

```diff
@@ -75,10 +74,6 @@
     {/if}
   </div>
   <div class="bg-[#DDD6FE]">
-    <img
-      src={query ? noResultsImages : emptyImage}
-      alt=""
-      class="aspect-auto"
-    />
+    <img src={query || emptyImage} alt="" class="aspect-auto" />
```

Comment:
How will this behave if there is a query? If we only want to show the `emptyImage` when there is no query, we should wrap the whole `<img />` in `{#if !query}`.

Reply:
Yeah we don't want to use the query as the src of the img, it will try to find it

<img width="604" alt="Screenshot 2025-07-02 at 9 07 17 AM" src="https://github.com/user-attachments/assets/3a13271b-c66d-4f9d-9251-f329c9ddc977" />